### DISCUSSION TITLE
9.6 upgrade weird color issue

### DISCUSSION THREAD

**Discussion by savager** [directus/directus#12182] [Archive]
I just upgraded directus to v9.6 and am seeing some kind of theming error. Half of the accent colors have turned a bright purple. I don't see any corresponding option to change this back, and I don't think I had it previously set. 

![image](https://user-images.githubusercontent.com/1455337/158461834-0a1a51be-8905-4604-9a74-56ab0fb04650.png)

**Comment by rijkvanzanten** [comment]
Not an error! The base theme was updated in https://github.com/directus/directus/pull/11952 in preparation of bigger custom themes support 👍🏻

**Comment by savager** [comment]
I'm so sorry, I just assumed it was a bug because it was so bright and looked harsh. My bad. 

I just saw this: https://github.com/directus/directus/issues/11985

**Reply by rijkvanzanten** [reply]
No problem 👍🏻
